helevents/tests/test_gdpr_api_delete.py

```diff
@@ -1,9 +1,10 @@
 from unittest.mock import patch
+from uuid import UUID
```

Comment:
Churn happening in this commit. Should combine small logical wholes into own commits => e.g introducing tests for these delete and get in one commit or their own (or even part of the commits where the get function is introduced and where the delete part is introduced)